src/Uno.UI/UI/Xaml/Media/Imaging/BitmapSource.cs

```diff
@@ -59,8 +59,18 @@ public void SetSource(Stream streamSource)
 		{
 			PixelWidth = 0;
 			PixelHeight = 0;
-
-			Stream = streamSource;
+			
+			if (streamSource != null)
```

Comment:
Code convention here: when you have a `else` block, please try to avoid a _not equal_ `!=` had write the _equal_ `==` first.